doc/nrf/app_dev/device_guides/nrf54l/kmu_provision.rst

```diff
@@ -0,0 +1,64 @@
+.. _ug_nrf54l_developing_provision_kmu:
+
+nRF54L15 KMU provisioning
+#########################
+
+The nRF54L15 DK is equipped with Hardware Key Management Unit (KMU), that requires provisioning when in use.
+The |NCS| provides a west command, ``ncs-provision``, allowing to upload keys to the device though the Serial Write Debug (SWD) interface.
+
+Prerequisites
+*************
+
+First, you must install the nrfprovision script:
+
+.. parsed-literal::
+   :class: highlight
+
+    pip install nrfprovision==0.9.0 --index-url https://files.nordicsemi.com/artifactory/api/pypi/nordic-pypi/simple
+
+
+Provisioning keys to the board
+******************************
+
+Before uploading keys, ensure that the SoC is unprovisioned. 
+If the SoC has been previously provisioned and you need to use a different set of keys, you must first erase the SoC with the recovery command:
+
+.. tabs::
+
+    .. group-tab:: erase using nrfutil
+
+      .. parsed-literal::
+         :class: highlight
+
+          nrfutil device erase
+
+    .. group-tab:: erase using nrfjprog
+
+      .. parsed-literal::
+         :class: highlight
+
+          nrfjprog --eraseall
+
+Once you have an unprovisioned SoC, upload keys to the board by running the following command:
+
+.. parsed-literal::
+   :class: highlight
+
+    west ncs-provision upload -s nrf54l15 -k ed25519.pem -k ed25519-1.pem -k ed25519-2.pem
```

Comment:
it is not clear how to generate these "PEM" files. Please add a link to page that tells how to do this, or add that information here.